community/examples/htcondor-pool.yaml

```diff
@@ -38,11 +39,6 @@ deployment_groups:
   - id: htcondor_install
     source: community/modules/scripts/htcondor-install
 
-  - id: htcondor_services
```

Comment:
Why was this module removed?

Reply:
It was added prior to the development of `test_enabled_apis` validation and it requires the user running Terraform to have the ability to enable services. There is a known issue that Terraform cannot guarantee that APIs are globally enabled after the API call to enable them is complete. So, the functionality isn't perfect and we have new functionality which has better guarantees. Should have noted that in the PR description but I think we are better without it.

Reply:
SG - That's what I figured, but wanted to verify it was intentional.